packages/manager/modules/support/src/tickets/new-ticket/issues-selector/issues-selector.controller.js

```diff
@@ -12,35 +18,45 @@ export default class SupportIssuesSelectorController {
     this.issueTypes = null;
     this.issueType = null;
     this.issue = null;
-  }
 
-  $onChanges(changes) {
-    if (changes.category || changes.serviceType || changes.parentIssue) {
-      this.issue = null;
-      this.issueType = null;
-      this.fetchIssueTypes();
-    }
+    this.isLoading = true;
+
+    return this
+      .fetchIssueTypes()
+      .then(() => {
+        this.isLoading = false;
+      });
   }
 
   fetchIssueTypes() {
-    this.isLoading = true;
-    return this.OvhApiSupport.v6().getIssueTypes({
-      category: get(this.category, 'id'),
-      issueTypeId: get(this.parentIssue, 'id'),
-      language: this.$translate.use(),
-      serviceType: get(this.serviceType, 'name'),
-    }).$promise.then((items) => {
-      this.issueTypes = items;
-      if (items.length === 1) {
-        [this.issueType] = items;
-        this.issue = this.issueType;
-      }
-    }).finally(() => {
-      this.isLoading = false;
+    return this.OvhApiSupport.v6()
+      .getIssueTypes({
+        category: this.category.id,
+        issueTypeId: this.parentIssue && this.parentIssue.id,
+        language: this.$translate.use(),
+        serviceType: this.serviceType && this.serviceType.name,
+      }).$promise
+      .then((issues) => {
+        this.issues = issues;
+
+        if (issues.length === 1) {
+          this.issue = head(issues);
+          this.onChange({ issue: this.issue });
+        }
+      });
+  }
+
+  onIssueChange() {
+    const currentIssue = this.issue;
+    this.issue = null;
```

Comment:
why do you have to set this.issue = null and then restore its value?

Reply:
probably related to: https://github.com/ovh/manager/pull/1669/files#r341130820

Reply:
Since this ticket is in `quality check ok` I will merge it.

Feel free to raise an Issue if you think that part could be enhanced.

Thanks!